protos/schedules.proto

```diff
@@ -0,0 +1,29 @@
+syntax="proto3";
```

Comment:
can you add some comments here about what fields are required (if any)?

Reply:
added here: https://github.com/dataform-co/dataform/pull/223/files#diff-f6013730658e3d5392d8da6698d03b66R18